frames/src/main/java/com/checkout/frames/screen/billingaddress/usecase/BillingAddressInputComponentStateUseCase.kt

```diff
@@ -0,0 +1,82 @@
+package com.checkout.frames.screen.billingaddress.usecase
+
+import com.checkout.base.mapper.Mapper
+import com.checkout.base.usecase.UseCase
+import com.checkout.frames.component.billingaddressfields.BillingAddressInputComponentsContainerState
+import com.checkout.frames.component.billingaddressfields.BillingAddressInputComponentState
+import com.checkout.frames.screen.manager.PaymentStateManager
+import com.checkout.frames.style.component.billingformdetails.BillingAddressInputComponentStyle
+
+internal class BillingAddressInputComponentStateUseCase(
+    private val billingAddressInputComponentStateMapper: Mapper<BillingAddressInputComponentStyle,
+            BillingAddressInputComponentState>,
+    private val paymentStateManager: PaymentStateManager
+) : UseCase<List<BillingAddressInputComponentStyle>, BillingAddressInputComponentsContainerState> {
```

Comment:
As a suggestion, to make it more readable (without generics in generics :face_with_spiral_eyes:) we could create `BillingAddressInputComponentsContainerStyle` as a counterweight to `BillingAddressInputComponentsContainerState`.

Reply:
Btw could you please remind me, why we can't use only one input style and just add all other needed data to the global billing address style?

Reply:
Yes sure, as we discussed earlier with Alex (IOS). In IOS there is a possibility to change each input field style. For instance, I can provide different styling for phone, addressline2 and addressline1.